src/main.spec.js

```diff
@@ -97,6 +97,23 @@ test('purgeQueue deletes messages from queue', async () => {
   expect(message).toEqual(false);
 });
 
+test('default exchange', async () => {
+  const connection = await amqp.connect('some-random-uri');
+  const channel = await connection.createChannel();
+  await channel.assertQueue('retry-queue-10s');

```

Comment:
Whould be betrer to use test related queue names. Prefix `retry-` already used by tests below. So we may use something like `default-queue` instead.

Reply:
used declared functions